packages/storage/src/README.md

```diff
@@ -0,0 +1,45 @@
+[//]: # `{ "title": "upload", "order": 0.1 }`
+
+# Upload in the Browser
+
+It's possible to use [File](https://developer.mozilla.org/en-US/docs/Web/API/File) to upload a file. 
+
+
+Example usage of storage method in a hypothetical React component:
+{% code title="StorageUI.ts" overflow="wrap" lineNumbers="true" %}
+
```

Comment:
Maybe adding a bit of a description here of what the method is doing and what it returns would be cool for people who don't know what is happening.

Reply:
That's good will do that.